src/components/dashboard/Rewards.js

```diff
@@ -44,6 +45,14 @@ const RewardsTab = props => {
     return () => store.set('loadingIndicator')({ loading: false })
   }, [])
 
+  const onPressOk = useOnPress(() => {
+    window.open(getRewardsPath(), '_blank')
+  }, [getRewardsPath])
+
+  const onDismiss = useOnPress(() => {
+    props.navigation.navigate('Home')
+  }, [props.navigation])
+
```

Comment:
```suggestion
  const onPressOk = useOnPress(() => window.open(getRewardsPath(), '_blank'), [getRewardsPath])
  const onDismiss = useOnPress(() => props.navigation.navigate('Home'), [props.navigation])

```

Reply:
also please destructurize props in component's function declaration (e.g.  ({ navigation, <other props here> }) => )